cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -483,7 +484,7 @@ def create(self, catalog_item_class, *args, **kwargs):
             An instance of catalog_item_class
         """
         cat_item = self.instantiate(catalog_item_class, *args, **kwargs)
-        view = navigate_to(cat_item, 'Add', wait_for_view=0)
+        view = navigate_to(cat_item, 'Add', wait_for_view=10)
```

Comment:
This is the default wait_for_view value, so this isn't _needed_. But if you want to explicitly set it that's just fine.